examples/basic/src/VideoPlayer.tsx

```diff
@@ -106,7 +106,9 @@ const VideoPlayer: FC<Props> = ({}) => {
   }, [goToChannel, srcListId]);
 
   useEffect(() => {
-    NavigationBar.setVisibilityAsync('visible');
+    if (Platform.OS === 'android') {
```

Comment:
We can use the variable `isAndroid` instead of `Platform.OS === 'android'` throughout the entire app, or use `isIos` where applicable.

Reply:
Right, This is done, thank you !